CITATION.cff

```diff
@@ -2,8 +2,8 @@ cff-version: 1.2.0
 message: If you use this software, please cite it using the metadata from this file.
 type: software
 title: 'mrremind: MadRat REMIND Input Data Package'
-version: 0.176.1
-date-released: '2024-03-01'
+version: 0.176.1.9001
```

Comment:
Don't forget to update to non-development version number

Reply:
Are those not rejected in the merge?

Reply:
```
if (length(unlist(thisVersion)[-(1:3)])) {
    stop()
}
```